### PR TITLE
Update readme with screenshots

### PR DESCRIPTION
**Pull request by blaiseIT**
I plan to re-do the screenshots to get rid of the transparent space around them. Any other suggestions?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,60 @@
-# Olympus Backgammon – frontend
+# Olympus Backgammon
 
-React frontend for the work-in-progress Olympus Backgammon project. Connects to a separate [backend](https://github.com/michaelti/olympus-backgammon-backend) Node.js server.
+Portes, Plakoto, and Fevga are three Greek variants of backgammon commonly played together as Tavli. Olympus Backgammon allows players to play these games against their friends online or against a computer. Games can be setup in seconds at [olympus-backgammon.com](https://olympus-backgammon.com) on desktop or on mobile.
```
Comment (michaelti) [inline README.md:3]: "setup" -> "set up"

File: README.md [hunk 2]
```diff
@@ -1,17 +1,60 @@
...
+### Plakoto
+
+[How to play Plakoto](https://www.bkgm.com/variants/Plakoto.html)
+
+<p align="center">
+  <img src="./resources/plakoto-progress.png" alt="Plakoto game in progress" width="738">
+</p>
+
+### Fevga
+
```
Comment (michaelti) [inline README.md:43]: Add a screenshot for Fevga? Or tbh we might not need a screenshot for all three variants, up to you!

File: README.md [hunk 3]
```diff
@@ -1,17 +1,60 @@
...
+# Olympus Backgammon
 
-React frontend for the work-in-progress Olympus Backgammon project. Connects to a separate [backend](https://github.com/michaelti/olympus-backgammon-backend) Node.js server.
+Portes, Plakoto, and Fevga are three Greek variants of backgammon commonly played together as Tavli. Olympus Backgammon allows players to play these games against their friends online or against a computer. Games can be setup in seconds at [olympus-backgammon.com](https://olympus-backgammon.com) on desktop or on mobile.
 
 ![reviewdog](https://github.com/michaelti/olympus-backgammon-frontend/workflows/reviewdog/badge.svg)
 
-## Development
+<p align="center">
+  <img src="./resources/portes-start.png" alt="Portes new game" width="738">
```
Comment (michaelti) [inline README.md:8]: Do the screenshots have to be stored in this repo, or can they be hosted on imgur or something?

Comment (blaiseIT) [reply README.md:8]: They can be anywhere. What's your preference?

Comment (michaelti) [reply README.md:8]: Somewhere external I think. But no biggie either way.

File: README.md (hunk 2 above)
Comment (blaiseIT) [reply README.md:43]: Added but now the URL is off centre 😡

**Review by michaelti — APPROVED** [review]

**Pull request merged**